recipes/zyre/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+from sys import platform
+import re
+import os
+
+
+class ZyreConan(ConanFile):
+    name = "zyre"
+    version = "2.0.1"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zeromq/zyre"
+    description = "Local Area Clustering for Peer-to-Peer Applications."
+    topics = ("conan", "zyre", "czmq", "zmq", "zeromq",
+              "message-queue", "asynchronous")
+    exports_sources = ['CMakeLists.txt', 'Findzyre.cmake']
```

Comment:
conan has generators for this purpose.
Check out the [cmake_find_package generator](https://docs.conan.io/en/latest/reference/generators/cmake_find_package.html).
By adding this generator to a consumer of this recipe, it can also just do `find_package(zyre REQUIRED)`.

```suggestion
    exports_sources = "CMakeLists.txt"
```